packages/blob/src/copy.ts

```diff
@@ -8,17 +8,6 @@ import {
   validateBlobApiResponse,
 } from './helpers';
 
-export interface CopyCommandOptions extends BlobCommandOptions {
```

Comment:
Perhaps we should do this in a different PR?

And I think it's still valuable to have different interfaces for every command. So we can branch when needed. We should think about these types as public APIs (they are).

There should be a PutCommandOptions and a CopyCommandOptions that are separate on the public API level. Then internally they can be built via `extends` if necessary so we keep things DRY when doable.